src/msg.c

```diff
@@ -1065,7 +754,1282 @@ void draw_message_window(MessagePrintState* printer) {
     }
 }
 
-INCLUDE_ASM(void, "msg", appendGfx_message, MessagePrintState* printer, s16 arg1, s16 arg2, u16 arg3, u16 arg4, u16 arg5, u8 arg6);
+void appendGfx_message(MessagePrintState* printer, s16 posX, s16 posY, u16 additionalOffsetX, u16 additionalOffsetY,
+                       u16 flag, u8 alpha) {
+    SpriteRasterInfo sp50;
+    u16 sp80bool;
+    s16 sp8E;
+    s16 sp96;
+    s16 sp9E;
+    s16 palette;
+    s16 spAE;
+    u16 spB6;
+    u8 spB8;
+    f32 spBC;
+    s32* spC0;
+    MessageCharset* msgCharset;
+    f32 temp_f10;
+    f32 temp_f20;
+    f32 temp_f20_3;
+    f32 temp_f22_2;
+    f32 temp_f24;
+    f32 bubbleX;
+    f32 bubbleY;
+    f32 temp_f28;
+    f32 temp_f30;
+    f32 temp_f8;
+    f32 temp_f8_5;
+    f64 temp_f0;
+    f64 temp_f0_2;
+    f64 temp_f2_2;
```

Comment:
These three temps and temp_f2 can be changed to f32 as Ethan and I discussed in Discord.

Reply:
Changing these didn't work for me